doc/topics/releases/neon.rst

```diff
@@ -212,3 +212,13 @@ Module Deprecations
     - Support for the ``ssh.recv_known_host`` function has been removed. Please use the
       :py:func:`ssh.recv_known_host_entries <salt.modules.ssh.recv_known_host_entries>`
       function instead.
+
+State Deprecations
+------------------
+
+- The :py:mod:`win_servermanager <salt.states.win_servermanager>` state has been
+  changed as follows:
+
+    - Support for the ``force`` kwarg has been removed from the
```

Comment:
IMHO, we need to note the option that it has been replaced with, per the deprecation warning notice.

Reply:
@cachedout It hasn't been replaced by anything. This option hasn't actually been used for a while. All this PR does is remove the warning and make it officially removed.

Reply:
Oh actually you're right. I totally missed that notice. I'll fix it right now. :)